docs/api-center-analyzer-integration-nodejs.md

```diff
@@ -0,0 +1,146 @@
+<!-- markdownlint-disable MD033 -->
+# API Center Analyzer Integration
+
+You can analyze your API documents using Visual Studio Code (Standalone) or Azure Portal (Server-Side). This section will guide you through the integration process.
+
+## Standalone Analyzer through Visual Studio Code
+
+> To use this standalone analyzer, you need to install the [API Center extension](https://marketplace.visualstudio.com/items?itemName=apidev.azure-api-center) in Visual Studio Code.
+
+1. Install dependencies.
+
+     ```javascript
+    cd nodejs/webapi && npm install
+    ```
+
+1. Run the sample app.
+
+    ```javascript
+    npm start
+    ```
```

Comment:
Before doing this, why don't we run the bundler, like `webpack`, first? I'm not sure whether it's a common practice using the bundler for the backend API app or not?